src/bitcoin-tx.cpp

```diff
@@ -531,10 +665,14 @@ static void MutateTx(CMutableTransaction& tx, const string& command,
         MutateTxDelOutput(tx, commandVal);
     else if (command == "outaddr")
         MutateTxAddOutAddr(tx, commandVal);
-    else if (command == "outdata")
-        MutateTxAddOutData(tx, commandVal);
+    else if (command == "outpubkey")
+        MutateTxAddOutPubKey(tx, commandVal);
+    else if (command == "outmultisig")
+        MutateTxAddMultiSig(tx, commandVal);
```

Comment:
nit: should probably be called `MutateTxAddOutMultiSig` to stay inline with the rest of the nomenclature.